### PR TITLE
Support mysql and fade out Cubrid

### PR DESCRIPTION
**Pull request by donggyu04**
#341

Support MySQL instead of Cubrid.
For cubrid fade out, the migration script from cubrid to mysql is provided.

### CODE REVIEW

File: ngrinder-controller/src/main/java/org/hibernate/dialect/MYSQLExDialect.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+/**
+ * @since 3.5.0
+ */
+public class MYSQLExDialect extends MySQLDialect {
+
+	public MYSQLExDialect() {
+		super();
+		registerColumnType( Types.VARCHAR, 65535, "text" );
+	}
+}
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/hibernate/dialect/MYSQLExDialect.java]: Is there no way to use default mysqldialect?

File: ngrinder-controller/src/main/java/org/ngrinder/infra/init/LockExDatabaseChangeLogGenerator.java [hunk 2]
```diff
@@ -81,28 +80,14 @@ public ValidationErrors validate(LockExDatabaseChangeLogStatement statement, Dat
...
+						+ database.escapeColumnName(liquibaseSchema,
+										database.getDatabaseChangeLogTableName(),
+										"LOCKED")
+						+ " = "
+						+ TypeConverterFactory.getInstance().findTypeConverter(database).getBooleanType()
+										.getFalseBooleanValue();
+
 		updateStatement.setWhereClause(whereClause);
 
 		return SqlGeneratorFactory.getInstance().generateSql(updateStatement, database);
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/infra/init/LockExDatabaseChangeLogGenerator.java]: I think we can take out these class.

**Comment by junoyoon** [comment]
LockServiceEx seems not necessary.

**Comment by junoyoon** [comment]
LGTM

**Comment by jeff526** [comment]
nGrinder 3.5.0 How to continue to use cubrid, why give up cubrid and choose Mysql?

**Comment by donggyu04** [comment]
@jeff526 

Cubrid became EOL.
You can migrate data to MySQL. [guide](https://github.com/naver/ngrinder/wiki/mysql-migration)

**Pull request merged**